features/step_definitions/user_steps.rb

```diff
@@ -148,40 +109,40 @@
 end
 
 Given(/^I have coauthored a work as "(.*?)" with "(.*?)"$/) do |login, coauthor|
-  author1 = FactoryGirl.create(:pseud, :user => User.find_by_login(login))
-  author2 = FactoryGirl.create(:pseud, :user => User.find_by_login(coauthor))
+  author1 = FactoryGirl.create(:pseud, user: User.find_by_login(login))
+  author2 = FactoryGirl.create(:pseud, user: User.find_by_login(coauthor))
   FactoryGirl.create(:work, authors: [author1, author2], posted: true, title: "Shared")
 end
 
 # WHEN
 
-When /^I follow the link for "([^\"]*)" first invite$/ do |login|		
-  user = User.find_by_login(login)		
-  invite = user.invitations.first		
-  step(%{I follow "#{invite.token}"})		
+When /^I follow the link for "([^"]*)" first invite$/ do |login|
```

Comment:
Why has this dropped the \ to be consistent ? 

The other regex's have it in the first string ( It does not need it but it is I feel more right ).